jfrog-cli/artifactory/commands/golang/go.go

```diff
@@ -71,7 +73,16 @@ func ExecuteGo(noRegistry bool, goArg, targetRepo, buildName, buildNumber string
 	}
 	err := goutils.RunGo(goArg)
 	if err != nil {
-		return err
+		if !noRegistry && strings.EqualFold(err.Error(), "404 Not Found") {
+			// Need to run Go without Artifactory to resolve all dependencies.
+			log.Debug("Got", err.Error(), "from", details.GetUrl()+"api/go/"+targetRepo+".", "Trying resolving directly and publishing the dependencies to Artifactory")
```

Comment:
I suggest changing this log message from Debug to Info. I also suggest rephrasing it a bit to be:
*log.Info("Received", err.Error(), "from Artifactory.", "Trying download the dependencies for the VCS and upload them to Artifactory...")*

Reply:
Done